homeassistant/components/mopeka/__init__.py

```diff
@@ -22,11 +24,14 @@
 type MopekaConfigEntry = ConfigEntry[PassiveBluetoothProcessorCoordinator]
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: MopekaConfigEntry) -> bool:
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Looks like this one came back 
```suggestion
async def async_setup_entry(hass: HomeAssistant, entry: MopekaConfigEntry) -> bool:
```

Reply:
I thought you wanted me to revert my change https://github.com/home-assistant/core/pull/122705#discussion_r1696014545